pkg/virt-controller/services/template.go

```diff
@@ -313,6 +313,15 @@ func computePodSecurityContext(vmi *v1.VirtualMachineInstance, seccomp *k8sv1.Se
 	return psc
 }
 
+func slirpNetworkBindingPluginImage(kvConfig *v1.KubeVirtConfiguration) string {
```

Comment:
Please squash this commit with the first one.

Reply:
Done